src/ASM/LR/ASMu2D.h

```diff
@@ -428,31 +428,25 @@ class ASMu2D : public ASMunstruct, public ASM2D
   //! \param[in] oldVars Gauss point variables associated with \a oldBasis
   //! \param[out] newVars Gauss point variables associated with this patch.
   //! \param[in] nGauss Number of Gauss points along a knot-span
-  bool transferGaussPtVars(const LR::LRSplineSurface* oldBasis,
-                           const RealArray& oldVars, RealArray& newVars,
-                           int nGauss) const;
+  virtual bool transferGaussPtVars(const LR::LRSpline* oldBasis,
```

Comment:
Think you need to use the same name on the basis variable in the header file as in the implementation file, to avoid doxygen spew, maybe `old_basis`? Remember then also to update the doxy comments.

Reply:
done